tests/app/config/services.yaml

```
@@ -21,9 +21,7 @@ services:
         parent: 'default.converter.with.cache'
         public: true
         arguments:
-            $factory: '@Neusta\ConverterBundle\Tests\Fixtures\Factory\PersonFactory'
-            $populators:
-                - '@Neusta\ConverterBundle\Tests\Fixtures\Populator\PersonNamePopulator'
+            $inner: '@default.converter'
```

Does Symfony force us to re-define the `$inner` argument here? Would be sad...

```suggestion
            $inner: '@.inner'
```